foreign_cc/make.bzl

```diff
@@ -52,64 +51,27 @@ def _create_make_script(configureParameters):
     ])
 
     make_commands = []
-
-    if not ctx.attr.make_commands:
-        for target in ctx.attr.targets:
-            make_commands.append("{make} -C $$EXT_BUILD_ROOT$$/{root} {target} {args}".format(
-                make = attrs.make_path,
-                root = root,
-                args = args,
-                target = target,
-            ))
+    for target in ctx.attr.targets:
+        make_commands.append("{make} -C $$EXT_BUILD_ROOT$$/{root} {target} {args}".format(
+            make = attrs.make_path,
+            root = root,
+            args = args,
+            target = target,
+        ))
 
     return create_make_script(
-        workspace_name = ctx.workspace_name,
-        tools = tools,
-        flags = flags,
         root = root,
-        user_vars = dict(ctx.attr.make_env_vars),
-        deps = ctx.attr.deps,
         inputs = inputs,
-        prefix = install_prefix,
         make_commands = make_commands,
     )
 
-def _get_install_prefix(ctx):
-    if ctx.attr.prefix:
-        return ctx.attr.prefix
-    return "$$INSTALLDIR$$"
-
 def _attrs():
     attrs = dict(CC_EXTERNAL_RULE_ATTRIBUTES)
+    attrs.pop("make_commands")
     attrs.update({
         "args": attr.string_list(
             doc = "A list of arguments to pass to the call to `make`",
         ),
-        "keep_going": attr.bool(
-            doc = (
-                "__deprecated__: To maintain this behavior, pass `-k` to the `args` attribute " +
-                "when not using the `make_commands` attribute."
-            ),
-            mandatory = False,
-            default = True,
-        ),
-        "make_commands": attr.string_list(
```

Comment:
How would we support needing different arguments per target? I can imagine some Makefiles existing that need different args passed to install than build but we're removing the escape hatch that would support that here. I don't have a concrete example of this though. Thoughts?

Reply:
I was thinking this would not be supported. I've yet to come across a case where two libraries are built by the same target but require different args (at least looking back at examples within these rules and other projects I'm familiar with). If this is not the case then the current API would definitely need a refactor. Do you do this in any of your projects?

Reply:
No, I don't do this; was more of a hypothetical use case question. Just wanted to highlight that we are removing an extension point but I guess if someone wants this sort of behaviour they can create a starlark rule that makes use of the framework themselves to support this.

Reply:
I'm happy to revisit this and add support if there's a large desire for it. But I feel in the common case that users can split their targets up with very little effort or negative impact if they want to use different arguments.